pkg/providers/types/provider/provider_types.go

```diff
@@ -0,0 +1,12 @@
+package provider
+
+type ProviderType string
+
+const (
+	ProviderTypeCloudFoundry ProviderType = "cloud foundry"
```

Comment:
is this constant still useful?

Reply:
I thought that It could be useful for easing future extension. 
I don't see it as a huge overengineered thing, but i can remove it if you want :)